docs/readme.wwwallet_setup.md

```diff
@@ -0,0 +1,138 @@
+## Prerequisites
+- Docker and Docker Compose installed on your machine.
+- An instance of Mysql greater or equal to version 8.
+
+## Setup Instructions
+To set up an instance of wwwallet, the source code of the frontend and backend must be cloned from the respective GitHub repositories.
+The frontend and backend can be run using Docker. Below are the steps to set up and run wwwallet.
+
+1. Clone the repositories:
+```bash
+git clone https://github.com/wwWallet/wallet-backend-server.git
+git clone https://github.com/wwWallet/wallet-frontend.git
+```
+
+2. Navigate to the backend directory and edit the file `config/config.template.ts` to set the environment variables according to your requirements. 
+Note that you will need to set: 
+- the host and port where the backend will be running.
+- the database connection details to connect to your Mysql instance.
+- and the notification system need to be disabled if no firebase subscription is available.
+
+This is an example configuration for a local hosted instance:
+```typescript
+export const config = {
+	url: "localhost",
+	port: "8002",
+	appSecret: "SERVICE_SECRET",
+	ssl: false,
+	db: {
+		host: "localhost",
+		port: "3306",
+		username: "root",
+		password: "changeme",
+		dbname: "test",
+	},
+	walletClientUrl: "WALLET_CLIENT_URL",
+	webauthn: {
+		attestation: "direct",
+		origin: "WEBAUTHN_ORIGIN",
+		rp: {
+			id: "WEBAUTHN_RP_ID",
+			name: "wwWallet demo",
+		},
+	},
+	alg: "EdDSA",
+	notifications: {
+		enabled: false,
+		serviceAccount: "firebaseConfig.json"
+	}
+}
+```
+
+3. Build and run the backend Docker container:
+```bash
+cd wallet-backend-server
+docker build -t wwwallet-backend .
+docker run -d -p 8002:8002 --name wwwallet-backend wwwallet-backend
```

Comment:
what about using a prebuilt wwwallet docker image and do any kind of overload using docker mount points?